include/inexor/vulkan-renderer/application.hpp

```diff
@@ -12,37 +13,69 @@
 
 namespace inexor::vulkan_renderer {
 
+namespace input {
+class KeyboardMouseInputData;
+}
```

Comment:
```suggestion
// forward declarations
namespace inexor::vulkan_renderer::input {
class KeyboardMouseInputData;
}

namespace inexor::vulkan_renderer {
```

Would be an alternative too

Reply:
Done.